docs/en/seatunnel-engine/rest-api-v2.md

```diff
@@ -384,15 +384,17 @@ When we can't get the job info, the response will be:
 
 #### Parameters
 
-> |         name         |   type   | data type |            description            |
-> |----------------------|----------|-----------|-----------------------------------|
+> | name                 |   type   | data type |            description            |
+----------------------|----------------------|----------|-----------|-----------------------------------|
```

Comment:
```suggestion
> |----------------------|----------|-----------|-----------------------------------|
```
The markdown format not right.
![image](https://github.com/user-attachments/assets/75580798-0cc3-4cea-a9bd-acf049faded9)